GitUI/ScriptsEngine/SimplePrompt.Designer.cs

```diff
@@ -46,41 +47,42 @@ private void InitializeComponent()
             // 
             // txt_UserInput
             // 
+            txt_UserInput.Anchor = AnchorStyles.Top | AnchorStyles.Left | AnchorStyles.Right;
             txt_UserInput.Location = new Point(12, 21);
             txt_UserInput.Name = "txt_UserInput";
-            txt_UserInput.Size = new Size(257, 20);
+            txt_UserInput.Size = new Size(310, 23);
             txt_UserInput.TabIndex = 1;
+            txt_UserInput.KeyUp += txt_UserInput_KeyUp;
             // 
-            // label1
+            // labelInput
             // 
-            label1.AutoSize = true;
-            label1.Location = new Point(10, 5);
-            label1.Name = "label1";
-            label1.Size = new Size(127, 13);
-            label1.TabIndex = 2;
-            label1.Text = "Please specify your input:";
+            labelInput.AutoSize = true;
+            labelInput.Location = new Point(10, 5);
+            labelInput.Name = "labelInput";
+            labelInput.Size = new Size(141, 15);
+            labelInput.TabIndex = 2;
+            labelInput.Text = "Please specify your input:";
             // 
             // SimplePrompt
             // 
             AcceptButton = btn_OK;
             AutoScaleDimensions = new SizeF(96F, 96F);
             AutoScaleMode = AutoScaleMode.Dpi;
-            ClientSize = new Size(281, 73);
-            Controls.Add(label1);
+            ClientSize = new Size(334, 76);
+            Controls.Add(labelInput);
             Controls.Add(txt_UserInput);
             Controls.Add(btn_OK);
             Name = "SimplePrompt";
-            Text = "SimplePrompt";
+            Text = "User input";
             Shown += SimplePrompt_Shown;
             ResumeLayout(false);
             PerformLayout();
-
         }
 
         #endregion
 
         private Button btn_OK;
         private TextBox txt_UserInput;
```

Comment:
Please rename these and adjust the handles names
```suggestion
        private Button btnOk;
        private TextBox txtUserInput;
```